mne/io/tests/test_raw.py

```diff
@@ -972,3 +972,12 @@ def test_get_data_tmin_tmax():
 
     with pytest.raises(TypeError, match="tmax must be .* float"):
         raw.get_data(tmax=[1, 2])
+
+
+def test_resamp_noop():
+    """Tests resampling doesn't affect data if sfreq is identical."""
+    fname = Path(__file__).parent / "data" / "test_raw.fif"
```

Comment:
This can just use `raw_fname` defined at the top of the file. Not sure why the test above this doesn't 🤷